### ISSUE TITLE
Component with UseAnimationFrameLoop rerenders 

### ISSUE THREAD

**Issue by rafaferry** [EricLambrecht/react-timing-hooks#25]
Hello,

I'm doing some proof of concept using UseAnimationFrameLoop, but something is bothering me.
Whenever I have a component with useAnimationFrameLoop, it rerenders on every frame. Is this normal?

Here's an example:

```
import React, { useState } from 'react';
import { useAnimationFrameLoop } from 'react-timing-hooks';

const Test = () => {
  const [count, setCount] = useState(0);
  const [stop, setStop] = useState(false);

  console.count('videoblur');

  useAnimationFrameLoop(() => {
    setCount(count + 1);
  }, stop);

  return (
    <div>
      <p>{count}</p>
      <button onClick={() => setStop(!stop)}>Stop counting</button>
    </div>
  );
};

export default Test;
```
When this code runs, console.count('videoblur') keeps increasing indefinitely, even when i click on the stop button.

**Comment by EricLambrecht** [comment]
Hey @rafaferry ,

thank you for opening and issue!

Does the console output stop, if you remove the `useAnimationFrameLoop` or does it still output `videoblur` indefinetely.

If this changes nothing, than it's probably a parent component causing this issue. But if it then stops, this is probably a bug and I will investigate it! Thanks :)

**Comment by EricLambrecht** [comment]
I investigated it and found out that a component that is rendered normally (once) without the hook, is rendered in an infinite loop with the `useAnimationFrameLoop()` hook, so it is definitely a bug. I will try to find the cause and then deploy a fix as soon as posisble.

**Comment by EricLambrecht** [comment]
@rafaferry The issue has been fixed in version **2.2.3**, which is now live. Thanks for reporting!
